src/main/java/api/ProductQuantity.java

```diff
@@ -0,0 +1,24 @@
+package api;
+
+public class ProductQuantity {
```

Comment:
rename the dto, it could be AddToCartPayload, CartPayload or smth which reflects the gist of using this class